acq4/devices/AxoPatch200/AxoPatch200.py

```diff
@@ -12,29 +12,16 @@
 from acq4.util.debug import printExc
```

Comment:
These changes should be removed from this branch. You can use `rebase -i` and skip the merge commit, or I'll do it after the PR is ready to merge.